addons/l10n_de/report/din5008_report.xml

```diff
@@ -45,6 +45,27 @@
                 <div class="article din" t-att-data-oe-model="o and o._name" t-att-data-oe-id="o and o.id">
                     <div class="header_address">
                         <t t-if="address">
+                            <div class="company_invoice_address">
+                                <t t-if="company.street">
```

Comment:
are you sure this needs to be company.street?

Reply:
I changed that